### DISCUSSION TITLE
Is it possible to sort a Trakt list prior to creating Plex playlist?

### DISCUSSION THREAD

**Discussion by r3dlobst3r** [Taxel/PlexTraktSync#1554] [Q&A] [answered]
I have some lists that are for movies that I would like to sync, but when I do, they are sorted by rank. I am looking to get them imported by release date. I was looking at the Trakt api and it looks like you can use X-Sort-By and X-Sort-How but I am unsure how I can leverage that with PlexTraktSync

**Comment by jrpetersjr** [comment, marked as answer]
Have you tried sorting them on Trakt and then letting the sync run? Rank is the default, but you can change the sort type and order.

**Reply by simonc56** [reply]
The script sort by rank because it's plexapi default.
To change that, you'll need to update the PlexTraktSync code to leverage X-Sort-By and X-Sort-How as you said.

https://github.com/Taxel/PlexTraktSync/issues/54
https://trakt.docs.apiary.io/#reference/lists/list-items/get-items-on-a-list

https://github.com/Taxel/PlexTraktSync/blob/7e704e5feccc0067544f772e61d54621acd10b03/plextraktsync/trakt_list_util.py#L13-L16
